src/wp-includes/rest-api.php

```diff
@@ -1254,6 +1255,31 @@ function rest_add_application_passwords_to_index( $response ) {
 	return $response;
 }
 
+/**
+ * Adds the default template types and template part areas to the REST API index.
+ *
+ * @since 6.8.0
+ *
+ * @param WP_REST_Response $response REST API response.
+ * @return WP_REST_Response Modified REST API response.
+ */
+function rest_add_templates_default_data_to_index( $response ) {
```

Comment:
Removed inline type hint; this matches the `rest_add_application_passwords_to_index`.